custom_student_verification/admin.py

```diff
@@ -36,5 +40,60 @@ def get_queryset(self, request):
         queryset = queryset.prefetch_related('user__profile')
         return queryset
 
+    def update_fields_for_queryset(self, queryset, field, value):
+        """
+        Given a queryset, this method will update and save a field with a new value on all its records or rows.
+        """
+        successfully_updated_counter = 0
+        failed_to_update_counter = 0
+        for record in queryset:
+            try:
+                setattr(record, field, value)
+                record.save()
+                successfully_updated_counter += 1
+            except IntegrityError as e:
```

Comment:
Nit: Why catch `IntegrityError` error here? Can you add a comment as to why catch this?